### ISSUE TITLE
`linter-xo` does not report problems

### ISSUE THREAD

**Issue by PopGoesTheWza** [SamVerschueren/vscode-linter-xo#73]
I have the latest `xo` cli installed and working from the terminal. `package.json` has `xo` settings:

```json
  "xo": {
    "envs": [
      "node"
    ],
    "extends": "xo-typescript",
    "extensions": [
      "ts"
    ],
    "space": 2
  }
```

But inside vscode, does not report anything to the 'problems' viewport, and invoking 'Fix all auto-fixable problems' from  the command palette does not fix anything.

**Comment by papb** [comment]
I had the same problem, I just found out that you *must* set the `"xo.enable"` in your VSCode settings:

```json
{
  "xo.enable": true
}
```

...regardless of what's in your package.json.

**Comment by PopGoesTheWza** [comment]
@papb I had it set globally in vscode... with no luck... yet.

**Comment by papb** [comment]
@PopGoesTheWza Really? Weird... Good luck... Perhaps try reinstalling extensions, or even reinstalling VSCode itself, I don't know...

**Comment by PopGoesTheWza** [comment]
With no clear explanation, it finally started to report problems into the vscode Problems window.

It suddenly "kicked-in" during an hours long vscode session. I had `xo` already setup (but no reporting in vscode)

- `xo@0.27.2` installed both globally and as local dependency in my project folder
- `vscode-linter-xo` extension installed and enabled in vscode global settings
- `.xo-config.json`

My `.xo-config.json`:

```json
{
  "ignores": [".js"],
  "space": 2,
  "prettier": true
}
```

Now there's still something bothering me, though it may be WAD, but problems reported do not offer option for quick-fix nor  to open the rule documentation like `eslint` and `tslint` do.

**Comment by yakov116** [comment]
@PopGoesTheWza still have an issue?
